pkg/policy/mapstate.go

```diff
@@ -32,24 +32,42 @@ import (
 type Key = policyTypes.Key
 type Keys = policyTypes.Keys
 
+type MapStateMap map[Key]MapStateEntry
+
+// NewKey returns an ingress key for the given parameters
+func NewKey(direction uint8, identity uint32, proto uint8, port uint16, prefixLen uint8) Key {
```

Comment:
Suggestion: could we use a builder interface? something like

`IngressKey().TCPPort(80).Identity(42)`

Reply:
Would `WithIdentity(42)` be clearer?

Reply:
Added a new commit with this change to the end.